gradle/maven/defaults-maven.gradle

```diff
@@ -102,29 +123,74 @@ configure(subprojects.findAll { it.path in rootProject.published }) { prj ->
     gradle.projectsEvaluated {
       publishing {
         def configurePom = {
-          name = "Apache Solr/Lucene (${project.name})"
+          if (project.path.startsWith(":solr")) {
+            name = "Apache Solr (module: ${project.name})"
+            description = name
+            url = 'https://lucene.apache.org/solr/'
+          } else {
+            name = "Apache Lucene (module: ${project.name})"
+            description = name
+            url = 'https://lucene.apache.org/'
+          }
+
           licenses {
             license {
               name = 'Apache 2'
               url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
             }
           }
-        }
 
-        publications {
-          // JARS and sources, no javadocs (for local inspection only).
-          jars(MavenPublication) {
-            from components.java
-            groupId = project.group
-            artifactId = project.archivesBaseName
+          inceptionYear = "2000"
 
-            artifact sourcesJar
+          issueManagement {
+            system = "JIRA"
+            url = "https://issues.apache.org/jira/browse/LUCENE"
```

Comment:
This should be split by project?

Reply:
All of it should be project-dependent, really... But yes -- feel free to commit a patch.